Align protocol docs with style guide (#575)
LiskHQ/lisk-docs: modules/ROOT/pages/network.adoc

= Network

> :description: The network section of the Lisk protocol describes the Lisk P2P network used to exchange information such as blocks or transactions between nodes.

Reviewer: ```suggestion
:description: This section describes the Lisk P2P network used to exchange information such as blocks or transactions between nodes.
```
